addon/player.js

```diff
@@ -138,19 +180,30 @@ class Player extends BasePlayer {
 
     return this;
   }
+
+  destroy() {
+    // Clean up all the resources before destroy the player.
+    return new RSVP.Promise((resolve) => {
+      if($('#tsToolBar').length > 0) {
+        $('#tsToolBar').remove();
+      }
+
+      resolve();
+    });
+  }
 }
 
 // Singleton instance of Player
-let current = null;
+var tsCurrentPlayer = null;
```

Comment:
Variables inside ES6 modules are private to the module so it's not necesary to give it a globally unique name, just a module unique name, I guess `current` or `currentPlayer` is enough.